sys-libs/libnbd/libnbd-1.14.1.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit bash-completion-r1
+
+DESCRIPTION="NBD client library in userspace"
+HOMEPAGE="https://gitlab.com/nbdkit/libnbd"
+SRC_URI="https://download.libguestfs.org/libnbd/$(ver_cut 1-2)-stable/${P}.tar.gz"
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="fuse gnutls uri-support test"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	uri-support? ( dev-libs/libxml2 )
+	fuse? ( sys-fs/fuse:3 )
+	gnutls? ( net-libs/gnutls:= )
+"
+DEPEND="
+	${RDEPEND}
+	test? ( sys-block/nbd[gnutls?] )
+"
+BDEPEND="dev-lang/perl"
+
+src_prepare() {
+	default
+
+	# Some tests require impossible to provide features, such as fuse.
+	# These are marked by requires_... in the functions.sh shell
+	# library.  Rather than listing these tests, let's list out the
+	# impossible to support features and make them skip.
+	cat <<-EOF >> tests/functions.sh.in || die
+		requires_fuse ()
+		{
+			requires false
+		}
```

Comment:
That's only for style 😄 
```
requires_fuse () {
	requires false
}
```

Reply:
I picked consistency with the existing code  in that file